### PR TITLE
Add cairo-rs - Rust implementation of the Cairo VM

### PR DESCRIPTION
**Pull request by abdelhamidbakhta**
## Description

[cairo-rs](https://github.com/lambdaclass/cairo-rs) is a Rust implementation of the Cairo VM. Cairo is a programming language for writing provable programs, where one party can prove to another that a certain computation was executed correctly. Cairo and similar proof systems can be used to provide scalability to blockchains.

`cairo-rs` offers a very efficient Cairo virtual machine, and will be used by StarkNet sequencer.

### CODE REVIEW

**Review by Israel-Laguan — APPROVED** [review]
LGTM
image

File: README.md [hunk 1]
```diff
@@ -151,6 +151,7 @@ See also [Rust — Production](https://www.rust-lang.org/production) organizatio
 
 * [Akula](https://github.com/akula-bft/akula) - Rust Ethereum Execution Layer (EL) Client (WIP)
 * [Bitcoin Satoshi's Vision](https://github.com/brentongunning/rust-sv) [[sv](https://crates.io/crates/sv)] — A Rust library for working with Bitcoin SV .
+* [cairo-rs](https://github.com/lambdaclass/cairo-rs) - Rust implementation of the Cairo VM [![rust](https://github.com/lambdaclass/cairo-rs/actions/workflows/rust.yml/badge.svg)](https://github.com/lambdaclass/cairo-rs/actions/workflows/rust.yml)
```
Comment (Israel-Laguan) [inline README.md:154]:
```suggestion
* [cairo-rs](https://github.com/lambdaclass/cairo-rs) — Rust implementation of the Cairo VM [![rust](https://github.com/lambdaclass/cairo-rs/actions/workflows/rust.yml/badge.svg)](https://github.com/lambdaclass/cairo-rs/actions/workflows/rust.yml)
```

Small character change to be on the same as surrounding items

**Comment by abdelhamidbakhta** [comment]
> LGTM image

@Israel-Laguan  Workflow is blocked ?

**Comment by Israel-Laguan** [comment]
> @Israel-Laguan Workflow is blocked ?

Maybe an admin should restart manually, thanks for noting @abdelhamidbakhta

**Pull request merged**